commitlint.config.js

```diff
@@ -0,0 +1,9 @@
+const TYPE_ERROR = 2;
+
+/** @type {import('@commitlint/types').UserConfig} */
+export default {
+  extends: ["@commitlint/config-conventional"],
+  rules: {
```

Comment:
Good observation, but I think we can actually benefit from having all default choices (including "chore")